### ISSUE TITLE
Projects can not be opened from vrc-get-gui due to Unity license error

### ISSUE THREAD

**Issue by lu-na-r** [vrc-get/vrc-get#653]
### Description of the bug / バグの説明

When I open my Unity project from vpm-get-gui, Unity responds "License Error" although I own a valid license.
I can open the same project from Unity Hub without the error.

### Steps to Reproduce / 再現手順

Click "Open Unity" button

### Expected Behavior / 想定される挙動

Projects can be opened

### Screenshots / スクリーンショット

<img width="372" alt="image" src="https://github.com/vrc-get/vrc-get/assets/100114898/1dc02e42-c044-4bc7-9b03-445d2f7fe9fb">
<img width="1136" alt="image" src="https://github.com/vrc-get/vrc-get/assets/100114898/5fd6095f-f1dd-460b-a61b-39e392f09da9">

### Software Version / バージョン情報

v0.1.0-beta.15

### OS

macOS 14.3.1

### Additional context / 補足

I imagine that this is my environmental issue, but I don't know why this happens. That's why I open this issue, so in case of NTF, close this issue.

**Comment by anatawa12** [comment]
Can you open the unity from command line?

**Comment by lu-na-r** [comment]
So, what do you mean? Do you mean `vpm check unity`?

**Comment by anatawa12** [comment]
No.
We are able to launch unity with `/Applications/Unity/Hub/Editor/<version>/Unity.app/Contents/MacOS/Unity -projectPath /path/to/project` and vrc-get-gui internally use this to launch unity so could you try this manually?

**Comment by lu-na-r** [comment]
I have a different path so I executed `/Applications/2022.3.6f1/Unity.app/Contents/MacOS/Unity -projectPath /path/to/project` only to fail with the same error.

**Comment by lu-na-r** [comment]
Now I know this is my environmental issue rather than vrc-get-gui issue, but I have no idea why I suddenly had this error.

**Comment by anatawa12** [comment]
According to the installed path, it looks you have installed unity without unity hub. Did you installed the unity with Unity Hub or changed unity installation path in Unity Hub?

**Comment by lu-na-r** [comment]
> it looks you have installed unity without unity hub

No, I installed Unity with Unity Hub and I did not change the path either.

**Comment by lu-na-r** [comment]
I've solved this problem on my own. I found Unity license file (Unity_lic.ulf) did not exist at '/Library/Application\ Support/Unity' so I added my personal license with Unity Hub again, which enabled me to open my project with command (and vrc-get-gui); though I do not know why my license file disappeared at the directory nor it was vrc-get-gui related issue (I guess not). I apologise for opening this issue which is not estimably related to your project.

Now you can close this case with NTF.

**Comment by anatawa12** [comment]
I think it's better to detect the situation and suggest user to re-activate unity license so I keep this issue open.
